common/src/commonMain/kotlin/com/studo/campusqr/common/payloads/CheckInPayloads.kt

```diff
@@ -27,4 +27,9 @@ class CheckOutData(
 class LiveCheckIn(
   val activeCheckIns: Int,
   val qrCodeContent: String?
+) : ClientPayload
+
+@Serializable
+class AllActiveCheckIns(
```

Comment:
Naming seems weird because it implies that it contains all active check-ins. 
If this is for **requesting** all active check-ins, it should probably have a prefix like GetAllActiveCheckIns.

Reply:
Please check if there are other cases like "request payload has name that could imply response" that I didn't notice.

Reply:
Good suggestion 👍